python/ray/data/_internal/execution/operators/all_to_all_operator.py

```diff
@@ -20,6 +21,7 @@ def __init__(
         bulk_fn: AllToAllTransformFn,
         input_op: PhysicalOperator,
         num_outputs: Optional[int] = None,
+        sub_progress_bar_names: Optional[List[str]] = None,
```

Comment:
Should this continue to use stage / operator in name? Basically the `name` below is operator's name, and this list is sub-operator names. Mapping them to progress bars is internal.

Reply:
Given we don't have a concept of sub operator yet, I am thinking to introduce it later? This currently only serves as name of progress bar, so limit it scope for now. We add methods here `initialize_sub_progress_bars()` and `close_sub_progress_bars()`, so sub progress bar is not internal of `AllToAllOperator`, so I think having `sub_progress_bar_names` in `__init__` should be fine?